### PR TITLE
cloud learning path: add back up and restore

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

Add some documents into TiDB Cloud learning path:
- Add BR, Changefeed, and CLI into the "Use" section.
- Update the "Use" section to "Maintain".

image

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/_index.md [hunk 1]
```diff
@@ -37,19 +37,25 @@ hide_commit: true
...
-<LearningPath label="Use" icon="cloud7">
+<LearningPath label="Maintain" icon="cloud7">
 
 [Create a Cluster](https://docs.pingcap.com/tidbcloud/create-tidb-cluster)
 
 [Connect to a Cluster](https://docs.pingcap.com/tidbcloud/connect-to-tidb-cluster)
 
 [Use an HTAP Cluster](https://docs.pingcap.com/tidbcloud/tiflash-overview)
 
+[Back Up and Restore](https://docs.pingcap.com/tidbcloud/backup-and-restore)
```
Comment (qiancai) [inline tidb-cloud/_index.md:48]:
```suggestion
[Back Up and Restore Data](https://docs.pingcap.com/tidbcloud/backup-and-restore)
```

Comment (qiancai) [reply tidb-cloud/_index.md:48]: 之前左侧目录里也是写的 Back Up and Restore Data，后来有反馈说太长了，所以删除 Data，只保留了动词。右侧学习路径里的空间估计是够的

**Review by benmaoer — CHANGES_REQUESTED** [review]
LGTM

**Review by qiancai — APPROVED** [review]

**Review by benmaoer — APPROVED** [review]
LGTM

**Pull request merged**